pyprland/plugins/scratchpads/objects.py

```diff
@@ -39,6 +39,11 @@ def set_config(self, opts):
             opts["lazy"] = True
             if "match_by" not in opts:
                 opts["match_by"] = "class"
+        if state.hyprland_version < (0, 39, 0):
+            opts["close_special_workspace"] = True
+        elif opts.get("close_special_workspace", None) is None:
```

Comment:
elif not needed if you add the default value in get() at line 534

Reply:
I will update this